server/app.ts

```diff
@@ -3,6 +3,8 @@ import express from 'express'
 import dpsComponents from '@ministryofjustice/hmpps-connect-dps-components'
 
 import * as Sentry from '@sentry/node'
+// @ts-expect-error Import untyped middleware for cypress coverage
```

Comment:
There is a DT implementation apparently https://www.npmjs.com/package/@types/cypress__code-coverage - worth a try?

Reply:
We do have this in the package.json, but it still complains

Reply:
What a pain. Ok, no worries then.